fastlane/metadata/en-US/marketing_url.txt

```diff
@@ -1 +1 @@
-https://woocommerce.com/mobile/
\ No newline at end of file
+https://woocommerce.com/mobile/
```

Comment:
Must be a sneaky difference like CRLF at end of file… though if it were GH would put a ⛔️ icon to indicate that… so not sure what diff it's seeing…🤷‍♂️   Oh well at least that gives me an excuse to post some meme 😄 

![image](https://user-images.githubusercontent.com/216089/105075080-9151ab00-5a89-11eb-94bc-fe0a882dd4d1.png)